CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 Scalyr Agent 2 Changes By Release
 =================================
 
+## 2.1.3 "TBD" - April 30, 2020
```

Comment:
Hmm.. We shouldn't merge this commit until after the 2.1.3 release is cut.  Also, we have to be careful about releasing a version of the agent that supports these compression algorithms until the server-side is deployed to production.

Reply:
The idea is to keep changelog always up to date / in sync with all the changes (aka so before the release we just do a quick glance over it and make minor modifications).

The only reason I used 2.3.1 here is because of the build_package.py script which expects a header to be in a specific format.

Otherwise, I would just use ``## in development`` or similar and also skip "packaged by" line until we are ready for an actual release.

Having said that, why shouldn't we merge this commit before then? IIRC, we only build new staging / unstable packages on release branch via manual trigger.

Or am I missing something?